roofit/roofitcore/src/RooGlobalFunc.cxx

```diff
@@ -80,6 +79,36 @@ RooCmdArg processFlatMap(const char *name, Func_t func, Detail::FlatMap<Key_t, V
    return container;
 }
 
+int interpretString(std::string const &s)
+{
+   return gInterpreter->ProcessLine(s.c_str());
+}
+
+Color_t interpretColorString(std::string const &color)
+{
+   using Map = std::unordered_map<std::string, Color_t>;
+   // Color dictionary to define matplotlib conventions
+   static Map colorMap{{"r", kRed},   {"b", kBlue},  {"g", kGreen},   {"y", kYellow},
+                       {"w", kWhite}, {"k", kBlack}, {"m", kMagenta}, {"c", kCyan}};
+   auto found = colorMap.find(color);
+   if (found != colorMap.end())
+      return found->second;
+   // Use interpreter if color was not matched in the color map
+   return gInterpreter->ProcessLine(color.c_str());
```

Comment:
What happens if I ask for "myColorThatDoesNotExist"?

Reply:
With the `GetColorByName` that you suggested: it will return "-1", and then RooFit falls back to black, like suggested by @AmyWinehouse.